lib/podium.ex

```diff
@@ -28,38 +28,37 @@ defmodule Podium do
       |> inject_application_uid()
       |> inject_source_type()
 
-    conversation_item =
-      Caramelize.camelize(%{
-        conversation_item: conversation_item
-      })
+    params = %{
+      conversation_item: conversation_item
+    }
 
-    API.post("/conversation_items", Caramelize.camelize(conversation_item))
+    API.post("/conversation_items", Caramelize.camelize(params))
   end
 
   @doc """
   Update a [conversation item](https://hexdocs.pm/podium_ex/Podium.ConversationItem.html#content).
   """
   @spec update_conversation_item(ConversationItem.t()) :: ConversationItem.t()
-  def update_conversation_item(%ConversationItem{uid: uid} = item) do
+  def update_conversation_item(%ConversationItem{} = item) do
     conversation_item =
       item
       |> remove_nils()
       |> inject_application_uid()
       |> inject_source_type()
 
-    conversation_item = %{
+    params = %{
       conversation_item: conversation_item
     }
 
-    API.put("/conversation_items/#{uid}", Caramelize.camelize(conversation_item))
+    API.put("/conversation_items", Caramelize.camelize(params))
   end
 
   @doc """
   Delete a [conversation item](https://hexdocs.pm/podium_ex/Podium.ConversationItem.html#content).
   """
-  @spec delete_conversation_item(String.t()) :: :ok
-  def delete_conversation_item(uid) do
-    API.delete("/conversation_items/#{uid}")
+  @spec delete_conversation_item(String.t(), String.t(), String.t()) :: :ok
+  def delete_conversation_item(application_uid, organization_uid, uid) do
+    API.delete("/applications/#{application_uid}organizations/#{organization_uid}/conversation_items/#{uid}")
```

Comment:
Rather than pulling in the `application_uid` from a parameter, can we get it from the config like we do in the `inject_application_uid` function? And then I would say that the `organization_uid` should probably be the second parameter since the conversation item `uid` is the more identifying one here.